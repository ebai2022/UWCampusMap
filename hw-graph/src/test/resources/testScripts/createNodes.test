# Test node creation

# Test single case
CreateGraph A
AddNode A e1

# Test multi case
AddNode A e2
AddNode A e3