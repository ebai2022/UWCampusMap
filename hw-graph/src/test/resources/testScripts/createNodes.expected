# Test node creation

# Test single case
created graph A
added node e1 to A

# Test multi case
added node e2 to A
added node e3 to A